lib/msal-browser/src/app/ClientApplication.ts

```diff
@@ -209,11 +211,13 @@ export abstract class ClientApplication {
                 let redirectResponse: Promise<AuthenticationResult | null>;
                 if (request && NativeMessageHandler.isNativeAvailable(this.config, this.logger, this.nativeExtensionProvider) && this.nativeExtensionProvider && !hash) {
                     this.logger.trace("handleRedirectPromise - acquiring token from native platform");
+                    handleRedirectPromiseMeasurement = this.performanceClient.startMeasurement(PerformanceEvents.HandleRedirectPromiseMeasurement, request.correlationId);
```

Comment:
Since we aren't already collecting telemetry for redirect calls I'd suggest we discuss whether or not this adds value and consider what else we would want to collect, if any. If we do pick it up let's do so in a separate PR

Reply:
+1 for separating "redirect" telemetry in a separate PR. I do think it helps to add it, just not here.

Reply:
Addressed! Thanks